e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/AuthTest.groovy

```diff
@@ -31,21 +28,21 @@ class AuthTest extends Specification {
 
     def "a 400 response is returned when request has invalid format"() {
         given:
-        def invalidRequest= "%g"
+        def invalidRequest = "%g"
 
         when:
-        def response = Client.post(authEndpointPath, invalidRequest, ContentType.APPLICATION_FORM_URLENCODED)
+        def response = AuthClient.loginRaw(invalidRequest, "asdf")
 
         then:
         response.getCode() == 400
     }
 
     def "a 401 response is returned when poorly formatted request"() {
         given:
-        def invalidRequest= "invalid-request"
+        def invalidRequest = "invalid-request"
 
         when:
-        def response = Client.post(authEndpointPath, invalidRequest, ContentType.APPLICATION_FORM_URLENCODED)
+        def response = Client.post(authEndpointPath, invalidRequest, ContentType.APPLICATION_FORM_URLENCODED, Map.of())
```

Comment:
Could be written using the new method like this:
`def response = AuthClient.loginRaw(invalidRequest, "asdf")`

Reply:
I changed it to AuthClient.